app/components/oral_history/footnote_reference_component.rb

```diff
@@ -8,11 +8,12 @@ module OralHistory
   # The tooltip hover is based on:
   # http://hiphoff.com/creating-hover-over-footnotes-with-bootstrap/
   class FootnoteReferenceComponent < ApplicationComponent
-    attr_reader :footnote_text, :number
+    attr_reader :footnote_text, :number, :show_dom_id
 
-    def initialize(footnote_text:, number:)
+    def initialize(footnote_text:, number:, show_dom_id:)
```

Comment:
sorry to be nitpicking, but show_dom_id should be optional and default to true maybe? Meh, who cares, I'm gonna merge anyway.